src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,53 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  margin: auto ;
```

Comment:
Don't add margin to a block, only for element. Use mix.
 More details in video.
[video, timing 4:15](https://mate.academy/learn/css-advanced/bem#/video/911)

And you should display a calendar in the middle of the screen. Now it centred horizontally, but in the top of the screen(not middle).
![Screenshot 2022-07-22 at 15 29 26](https://user-images.githubusercontent.com/17098736/180439391-f29b8273-192c-45dc-92ec-da0b7794539d.png)